internal/sdkprovider/service/governance/governance_subscription.go

```diff
@@ -0,0 +1,256 @@
+		DeleteContext: common.WithGenClient(resourceGovernanceSubscriptionDelete),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: schemautil.DefaultResourceTimeouts(),
+		Schema:   aivenGovernanceSubscriptionSchema,
+	}
+}
+
+func resourceGovernanceSubscriptionCreate(ctx context.Context, d *schema.ResourceData, client avngen.Client) error {
+	var req governance.OrganizationGovernanceSubscriptionCreateIn
+
+	req.SubscriptionName = d.Get("subscription_name").(string)
+	req.SubscriptionType = governance.SubscriptionType(d.Get("subscription_type").(string))
+
+	req.SubscriptionData = governance.SubscriptionDataIn{
+		ProjectName: d.Get("subscription_data.0.project").(string),
+		ServiceName: d.Get("subscription_data.0.service_name").(string),
+	}
+
+	if ownerUserGroupID := d.Get("owner_user_group_id").(string); ownerUserGroupID != "" {
+		req.OwnerUserGroupId = &ownerUserGroupID
+	}
+
+	if username := d.Get("subscription_data.0.username").(string); username != "" {
+		req.SubscriptionData.Username = username
+	}
+
+	for _, v := range d.Get("subscription_data.0.acls").(*schema.Set).List() {
+		acl := v.(map[string]interface{})
+		req.SubscriptionData.Acls = append(req.SubscriptionData.Acls, governance.AclIn{
+			ResourceName:   acl["resource_name"].(string),
+			ResourceType:   governance.ResourceType(acl["resource_type"].(string)),
+			Operation:      governance.OperationType(acl["operation"].(string)),
+			PermissionType: governance.PermissionType(acl["permission_type"].(string)),
+			Host:           acl["host"].(string),
+		})
+	}
+
+	subscription, err := client.OrganizationGovernanceSubscriptionCreate(
+		ctx,
+		d.Get("organization_id").(string),
+		&req,
+	)
+	if err != nil {
+		return err
+	}
+
+	subscriptionData := make([]map[string]any, 1)
+	subscriptionData[0] = make(map[string]any, 1)
+	subscriptionData[0]["project"] = subscription.SubscriptionData.ProjectName
+	subscriptionData[0]["service_name"] = subscription.SubscriptionData.ServiceName
+	subscriptionData[0]["username"] = subscription.SubscriptionData.Username
+
+	acls := make([]map[string]string, len(subscription.SubscriptionData.Acls))
+	for i, acl := range subscription.SubscriptionData.Acls {
+		acls[i] = make(map[string]string, 1)
+		acls[i]["id"] = acl.Id
+		acls[i]["resource_name"] = acl.ResourceName
+		acls[i]["resource_type"] = string(acl.ResourceType)
+		acls[i]["pattern_type"] = string(acl.PatternType)
+		acls[i]["operation"] = string(acl.Operation)
+		acls[i]["permission_type"] = string(acl.PermissionType)
+		acls[i]["host"] = acl.Host
+	}
+	subscriptionData[0]["acls"] = acls
+
+	d.SetId(subscription.SubscriptionId)
+
+	return d.Set("subscription_data", subscriptionData)
+}
+
+func resourceGovernanceSubscriptionRead(ctx context.Context, d *schema.ResourceData, client avngen.Client) error {
+	subscription, err := client.OrganizationGovernanceSubscriptionGet(
+		ctx,
+		d.Get("organization_id").(string),
+		d.Id(),
```

Comment:
Likely, the `id` is a composite field: `org_id/sub_id`, because that's what you need to read the resource on `terraform import`.
The subscription id should be stored in a separate field, like `subscription_id`.